prototype/Frontend/user_replies.html

```diff
@@ -101,411 +78,335 @@
-              <li class="dropdown">
-                <a href="#">
-                  <span>Contributions</span>
-                  <i class="bi bi-chevron-down"></i>
+                <li>
+                    <a class="getstarted scrollto user-btn" href="index.html">
+                    <span><i class="fa-solid fa-house fs-6"></i></span>
+                    </a>
+                </li>
+                <!-- About Dropdown -->
+                <li class="dropdown">
+                    <a href="#">
+                    <span>About</span> <i class="bi bi-chevron-down"></i>
                 </a>
-                <ul>
-                  <li class="border-bottom"><a href="blogs.html">Blogs</a></li>
-                  <li class="border-bottom">
-                    <a href="search-weekly-challenges.html">Challenge</a>
-                  </li>
-                  <li>
-                    <a href="search_contributions.html">Search</a>
-                  </li>
-                </ul>
-              </li>
-              <!-- Team Dropdown -->
-              <li class="dropdown">
-                <a href="#">
-                  <span>Teams</span>
-                  <i class="bi bi-chevron-down"></i>
+                    <ul>
+                    <li class="border-bottom">
+                        <a href="about.html">About Us</a>
+                    </li>
+                    <li class="border-bottom">
+                        <a href="contact.html">Contact Us</a>
+                    </li>
+                    <li><a href="faq.html">FAQ</a></li>
+                    </ul>
+                </li>
+                <!-- Contributions Dropdown -->
+                <li class="dropdown">
+                    <a href="#">
+                    <span>Contributions</span>
+                    <i class="bi bi-chevron-down"></i>
                 </a>
-                <ul>
-                  <li class="border-bottom"><a href="team.html">Member</a></li>
-                  <li class="border-bottom"><a href="guest.html">Guest</a></li>
-                  <li><a href="champions.html">Champions</a></li>
-                </ul>
-              </li>
-              <!-- Reviews Dropdown -->
-              <li class="dropdown">
-                <a href="#">
-                  <span>Reviews</span>
-                  <i class="bi bi-chevron-down"></i>
+                    <ul>
+                    <li class="border-bottom"><a href="challenges.html">Challenges</a></li>
+
+                    <li class="border-bottom"><a href="blogs.html">Blogs</a></li>
+                    <li class="border-bottom"><a href="list-outreachy-applicants.html">Outreachy Applicants</a></li>
+                    <li ><a href="search_contributions.html">Search Contributions</a></li>
+
+                    </ul>
+                </li>
+                <!-- Team Dropdown -->
+                <li class="dropdown">
+                    <a href="#">
+                    <span>Team</span>
```

Comment:
Please stop pushing the old navbar back again ... update your navbar to the one recently commited